suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -725,15 +725,26 @@ export const getMainnets = (debug = false, bnb = false) =>
             (bnb || n.symbol !== 'bnb'),
     );
 
-export const getTestnets = (debug = false) =>
+export const getTestnetsCompatible = (debug = false) =>
     networksCompatibility.filter(
         n => !n.accountType && n.testnet === true && (!n.isDebugOnlyNetwork || debug),
     );
 
-export const getAllNetworkSymbols = () => networksCompatibility.map(n => n.symbol);
+const networksCollection: Network[] = Object.values(networks);
+
```

Comment:
`getMainnets` and `getTestnets` are new, but I have manually checked that the **output is the same** as the old versions, except the networks now contain `accountTypes: [...]`. So it's only extended, not limited.

Still - I had to create the `Compatible` functions because of interoperability with `networksCompatibility` - they are often used in conjunction, and TS would scream, even though in this case, there is nothing missing :see_no_evil: